plutus-core/plutus-core/src/PlutusCore/Default/Builtins.hs

```diff
@@ -1065,6 +1072,7 @@ instance Flat DefaultFun where
               MkPairData               -> 48
               MkNilData                -> 49
               MkNilPairData            -> 50
+              SerialiseData            -> 53
```

Comment:
why 53? because of the other PR? I think whichever one goes in first should take the first numbers, otherwise we have to remember what's going on in our heads